lerobot/common/policies/act/modeling_act.py

```diff
@@ -347,56 +357,55 @@ def forward(self, batch: dict[str, Tensor]) -> tuple[Tensor, tuple[Tensor, Tenso
                 batch["observation.state"].device
             )
 
-        # Prepare all other transformer encoder inputs.
-        # Camera observation features and positional embeddings.
-        all_cam_features = []
-        all_cam_pos_embeds = []
-        images = batch["observation.images"]
+        # Prepare transformer encoder inputs.
```

Comment:
FYI. I did some rearranging here that was not strictly necessary for this PR. I did it to keep things more structured now that I'm adding yet another input token. The goal was to start with these two lists `encoder_in_tokens` and `encoder_in_pos_embed` containing the minimal necessary tokens and then extend them with the needed tokens incrementally and in the right order.

Please check carefully.

Reply:
Yes, adding tokens is much clearer now, thanks!